zerver/models.py

```diff
@@ -1669,17 +1668,19 @@ def get_status_dict_by_realm(realm_id):
             'user_profile__enable_offline_push_notifications',
         )
 
-        mobile_user_ids = [row['user'] for row in PushDeviceToken.objects.filter(
+        mobile_query = PushDeviceToken.objects.filter(
```

Comment:
was this area also changed for semantic clarity?

Reply:
yep